samples/WOCCatalog/WOCCatalog/PickersViewController.m

```diff
@@ -129,20 +95,19 @@ - (void)action:(id)sender
     }
 }
 
-- (CGRect)calcFrameForPicker:(CGSize)size
+- (CGRect)calcFrameForPicker
 {
     CGRect screenRect = [[UIScreen mainScreen] applicationFrame];
```

Comment:
Several of these were just to maintain some of the existing code. But, fine, I'll update and resubmit.